### PR TITLE
Feat/181 extend calculations for having berm as part of input profile

### PR DESCRIPTION
**Pull request by ArdtK**
## Issue addressed
Solves #181 #233 #235

## Code of conduct
- [x] I HAVE NOT added sensitive or compromised (test) data to the repository.
- [x] I HAVE NOT added vulnerabilities to the repository.
- [ ] I HAVE discussed my solution with (other) members of the KOSWAT team.

## What has been done?
- Solution for bugs reported in #233 and #235
- Adjusted the input profile calculations for having a berm
- Updating the test references (images and cost csv)

### Checklist
- [x] Tests are either added or updated.
- [x] Branch is up to date with `master`.
- [ ] Updated documentation if needed.

## Additional Notes (optional)
Add any additional notes or information that may be helpful.

### CODE REVIEW

File: koswat/cost_report/io/summary/koswat_summary_exporter.py
Comment (ArdtK) [inline koswat/cost_report/io/summary/koswat_summary_exporter.py:1]: @peterdgr what is the reason for this change?

File: koswat/dike_reinforcements/input_profile/piping_wall/piping_wall_input_profile_calculation.py [hunk 1]
```diff
@@ -100,33 +72,118 @@ def _determine_construction_type(
...
+            base_data.polderside_berm_height - base_data.polderside_ground_level
         )
+        _berm_factor_old = _berm_height_old / _dike_height_old
+
+        if _berm_factor_old > soil_settings.max_berm_height_factor:
+            _berm_old_is_stability = True
+        else:
+            _berm_old_is_stability = False
+
+        _dikebase_stability_old = (
```
Comment (ArdtK) [inline koswat/dike_reinforcements/input_profile/piping_wall/piping_wall_input_profile_calculation.py:104]: @peterdgr what does "dikebase" stand for in the calculations?

File: koswat/cost_report/io/summary/koswat_summary_exporter.py
Comment (peterdgr) [reply koswat/cost_report/io/summary/koswat_summary_exporter.py:1]: I found some exceptions where generating the shapefile would crash. The ones I found had to do with the order strategy, these are fixed, but most of all I'd like to continue the analysis with the next dikesections even if generating of the shapefile fails.

File: koswat/dike_reinforcements/input_profile/vertical_piping_solution/vps_input_profile.py
[the remark this answers is not in the document]
Comment (peterdgr) [reply koswat/dike_reinforcements/input_profile/vertical_piping_solution/vps_input_profile.py:1]: @ArdtK Yes, this is intentional for now. In the future I would like to specify a percentage build/unbuild in the cost settings specific for each measure, I will make a separate issue for that
